akka-actor-typed/src/main/scala/akka/actor/typed/scaladsl/Routers.scala

```diff
@@ -21,8 +34,8 @@ object Routers {
    * before the group detects this. Because of this it is best to unregister routees from the receptionist and not stop
    * until the deregistration is complete to minimize the risk of lost messages.
    */
-  def group[T](key: ServiceKey[T]): GroupRouter[T] =
-    new GroupRouterBuilder[T](key)
+  def group[T](key: ServiceKey[T], localAffinity: Boolean): GroupRouter[T] =
```

Comment:
Maybe `localAffinity` parameter should be on the specific routing logic instead on all group routers level? For example I don't think it makes sense to allow localAffinity for consistent hashing.

The api could be in the `withRandomRouting` and `withRoundRobinRouting` methods instead.
```
Routers.group(key).withRoundRobinRouting(localAffinity = true)
```

Reply:
I wonder if it should be `preferLocalRoutees` as well, as that makes what it does more obvious and less academic-hard-wordy?

Reply:
sounds good to me